Cabal/doc/installing-packages.rst

```diff
@@ -200,7 +200,7 @@ This is just syntactic sugar for
 ::
 
     repository hackage.haskell.org
-      url: hackage.haskell.org:http://hackage.haskell.org/packages/archive
+      url: http://hackage.haskell.org/packages/archive
```

Comment:
Not sure about this URL, but this seems like copy-paste error based on the above.

Reply:
yes